crates/sui-faucet/src/faucet/simple_faucet.rs

```diff
@@ -823,9 +838,18 @@ mod tests {
         }
 
         faucet.retry_wal_coins().await.ok();
+        let mut wal = faucet.wal.lock().await;
+        assert!(!wal.log.is_empty());
+
+        // Set in flight to false so WAL will clear
+        wal.set_in_flight(coin_id, false)
+            .expect("Unable to set in flight status to false.");
```

Comment:
Clarifying: do we wanna panic here? If not can we map_err to include the error that made us unable to set in flight. And might be worth including the error if we do want to panic too

Reply:
This is a test and I just needed to handle the return type